### DISCUSSION TITLE
My-CS-Notebook/Gossip/%E5%86%99%E5%9C%A8%E8%AF%BB%E7%A0%94%E4%B8%80%E5%B9%B4%E4%B9%8B%E9%99%85/

### DISCUSSION THREAD

**Discussion by giscus** [Zhang-Each/My-CS-Notebook#5] [General]
# My-CS-Notebook/Gossip/%E5%86%99%E5%9C%A8%E8%AF%BB%E7%A0%94%E4%B8%80%E5%B9%B4%E4%B9%8B%E9%99%85/

https://zhang-each.github.io/My-CS-Notebook/Gossip/%E5%86%99%E5%9C%A8%E8%AF%BB%E7%A0%94%E4%B8%80%E5%B9%B4%E4%B9%8B%E9%99%85/

**Comment by brcarry** [comment]
🥺小角龙前辈qwq

**Comment by leo-934** [comment]
我想喝无糖可口可乐
